### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dilipchauhan14**
hjhjbbkjk

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # HelpDesk
 Helps you to waste your time.
```
Comment (sammarth-kapse) [inline README.md]: ok

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
 //scjnfvjknfd
+// bjjgyhjbhjb
```
Comment (dilipchauhan14) [inline README.md]: This line was added

Comment (sammarth-kapse) [inline README.md]: delete this line

Comment (sammarth-kapse) [inline README.md]: delete plese

**Pull request merged**